hibernate-core/src/main/java/org/hibernate/engine/spi/SessionImplementor.java

```diff
@@ -26,7 +25,8 @@
  * {@link org.hibernate.type.Type}, {@link EntityPersister}
  * and {@link org.hibernate.persister.collection.CollectionPersister} implementations.
  *
- * A Session, through this interface and SharedSessionContractImplementor, implements:<ul>
+ * A Session, through this interface and SharedSessionContractImplementor, implements:
+ * <ul>
```

Comment:
Minor so don't worry about it here, but in the future this was the preferred syntax for html lists